My project2/My project/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+
+var a = false;
+string[] newarray = new string[10];
+void Note()
+{
+    
+    Console.WriteLine("Please write  note");
+
+    string note = Console.ReadLine();
+    newarray[0] = note;
+    Console.WriteLine("Do you want see note?");
+    string user = Console.ReadLine();
+
+    if (user.ToLower() == "no" )
+    {
+        Console.Clear(); 
+    }
+    else if (user.ToLower() == "yes" || user.ToLower() == "yeah") 
+    {
+        
```

Comment:
We musn't leave empty else if statement, as in this case we don't need else_if at all and we can remove this